ci-operator/step-registry/hypershift/azure/run-e2e-self-managed/hypershift-azure-run-e2e-self-managed-commands.sh

```
@@ -36,6 +36,36 @@ check_e2e_flag() {
+    fi
+    echo "${pullspec}"
+}
+
+if [[ -n "${NODEPOOL_N1_VERSION:-}" ]]; then
+    resolved=$(resolve_release_image "${NODEPOOL_N1_VERSION}") && OCP_IMAGE_N1="${resolved}"
+fi
+if [[ -n "${NODEPOOL_N2_VERSION:-}" ]]; then
+    resolved=$(resolve_release_image "${NODEPOOL_N2_VERSION}") && OCP_IMAGE_N2="${resolved}"
+fi
+if [[ -n "${NODEPOOL_N3_VERSION:-}" ]]; then
+    resolved=$(resolve_release_image "${NODEPOOL_N3_VERSION}") && OCP_IMAGE_N3="${resolved}"
```

Shellcheck (0.11.0)

[warning] OCP_IMAGE_N3 appears unused. Verify use (or export if used externally).

(SC2034)